.circleci/config.yml

```diff
@@ -40,43 +25,22 @@ jobs:
           path: build/test-results
       - store_artifacts:
           path: build/reports/tests
-  build_docker:
-    executor: deployer
-    steps:
-      - checkout
-      - setup_remote_docker:
-          docker_layer_caching: true
-      - hmpps/create_app_version
-      - run:
-          name: Build container and publish to docker hub
-          command: |
-            docker build --rm=false . \
-               --build-arg BUILD_NUMBER=$APP_VERSION \
-               --tag "mojdigitalstudio/community-api:$APP_VERSION" \
-               --label "maintainer=info@digital.justice.gov.uk" \
-               --label "app.version=$APP_VERSION" \
-               --label "build.version=$APP_VERSION" \
-               --label "build.number=$CIRCLE_BUILD_NUM" \
-               --label "build.url=$CIRCLE_BUILD_URL" \
-               --label "build.gitref=$CIRCLE_SHA1"
-
-            # publish to docker hub as the tag
-            docker login -u $DOCKER_USERNAME -p $DOCKER_PASSWORD
-            docker tag mojdigitalstudio/community-api:$APP_VERSION mojdigitalstudio/community-api:latest
-            docker push mojdigitalstudio/community-api:$APP_VERSION
-            docker push mojdigitalstudio/community-api:latest
+      - persist_to_workspace:
```

Comment:
shouldn't need any of the persist to workspace

Reply:
agreed, removed